azure-pipelines.yml

```diff
@@ -24,13 +22,31 @@ jobs:
       versionSpec: '$(python.version)'
       architecture: 'x64'
 
-  - script: python -m pip install --upgrade pip && pip install -r requirements.txt
-    displayName: 'Install dependencies'
+  - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+    displayName: Add conda to PATH
 
   - script: |
-      pip install pytest
+      conda update conda
+      conda create --yes --quiet --name bluesky python=$(python.version)
+    displayName: Create Anaconda environment
+
+  - script: |
+      conda activate bluesky
+      conda install -c anaconda qt sqlite
+      conda install matplotlib pyqt
+    displayName: Install Anaconda packages
+
+  - script: |
+      python -m pip install --upgrade pip
+      pip install -r requirements.txt
```

Comment:
This line isn't needed; our `setup.py` parses `requirements.txt` and installs them, so `pip install .` is sufficient.

Reply:
removed